src/main/kotlin/no/nav/hjelpemidler/service/oebsdatabase/HjelpemiddeloversiktDao.kt

```diff
@@ -44,6 +45,25 @@ class HjelpemiddeloversiktDao(private val dataSource: DataSource = Configuration
         return berikOrdrelinjer(items)
     }
 
+    fun utlånPåIsokode(fnr: String, isokode: String): List<Row> {
+        @Language("OracleSQL")
+        val query =
+            """
+            SELECT KATEGORI3_NUMMER  
+            FROM XXRTV_DIGIHOT_HJM_UTLAN_FNR_V
+            WHERE FNR = ?
+            AND KATEGORI3_NUMMER = ?
+            ORDER BY UTLÅNS_DATO DESC
+            """.trimIndent()
+
+        val items = sessionOf(dataSource).use {
+            it.run(
+                queryOf(query, fnr, isokode).map{row ->  row}.asList
```

Comment:
Hmm, litt usikker på dette med å returnere en Database Row-klasse fra en funksjon om lukker session den var åpnet fra. Hvis noen gjenbruker koden og faktisk prøver å hente ut detaljer (kollonner), vil den feile da? Eller holder Row-klassen alle data og er bare en ren dataklasse?

Reply:
Foreslår at man bare returnerer en List<String> for klarhet uansett siden du bare henter en kollonne og da må ikke caller-funksjonen vite hva den heter.

Reply:
`List<String>`

Reply:
```
queryOf(query, fnr, isokode).map{ row ->  row.string("KATEGORI3_NUMMER") }.asList
```

Reply:
Det kan godt hende du har rett, men vi pleier i hvertfall ikke ellers å returnere database-typen ut i apiet vårt. Alle andre plasser så returnerer vi grunnleggende typer som String/Int (eventuelt en liste av de), eller så oversetter vi til vår egen komplekse type i den map-funksjonen :)

Reply:
Mange eksempler her: https://github.com/navikt/hm-soknadsbehandling-db/blob/main/src/main/kotlin/no/nav/hjelpemidler/soknad/db/db/S%C3%B8knadStore.kt

Reply:
Gode poeng, kjørte en refactor nå.